CSV.js

```diff
@@ -195,6 +196,16 @@ function getValue(item, field) {
 			}
 			value += escapeValue(notes.join(valueSeparator));
 		break;
+		case 'date':
+			if (item.date) {
+				var dateISO = ZU.strToISO(item.date);
```

Comment:
Should not be performing this if `normalizeDate` is `false`. `var dateIso = normalizeDate ? .... : null;`